htdocs/societe/class/societe.class.php

```diff
@@ -3754,7 +3754,13 @@ function getOutstandingBills($mode='customer')
 				$tmpobject=new Facture($this->db);
 			}
 			while($obj=$this->db->fetch_object($resql)) {
-				$tmpobject->id=$obj->rowid;
+                $tmpobject->id=$obj->rowid;
+
+                $paiement = $tmpobject->getSommePaiement();
```

Comment:
This part of code was before after the 
if ($obj->paye == 0
Because if we forced manually the invoice to pay, it means we dont expect to have it paid and it should not be included into a remain to pay.